extra-network/zerotier-one/autobuild/defines

```diff
@@ -1,5 +1,6 @@
 PKGNAME="zerotier-one"
 PKGDES="ZeroTier creates secure networks between on-premise, cloud, desktop, and mobile devices"
 PKGSEC=net
-PKGDEP="gcc-runtime miniupnpc"
-BUILDDEP="ronn"
+PKGDEP="llvm-runtime miniupnpc cargo"
```

Comment:
It probably doesn't depend on `llvm-runtime` and `cargo`, right?

Reply:
After my test, yes.

Reply:
I have made an improvement.